### DISCUSSION TITLE
Quasar Chips?

### DISCUSSION THREAD

**Discussion by chrschorn** [zauberzeug/nicegui#2935] [Ideas / Feature Requests]
Hey, are you open to adding https://quasar.dev/vue-components/chip/ ? I might have time to work on this.

**Comment by falkoschindler** [comment]
Great idea, @chrschorn! That would be a nice addition to NiceGUI's UI elements. 🙂

**Reply by chrschorn** [reply]
Here we go 🙂 https://github.com/zauberzeug/nicegui/pull/2942
